lunch-application/src/theme/EditTopFive.vue

```diff
@@ -0,0 +1,116 @@
+        <tr>
+          <td>
+            <label id="choiceLabel" for="firstChoice">First Choice</label>
+          </td>
+          <td>
+            <select id="firstChoice" class= "form-control">
+              <option v-for="(data, index) in restaurantData" :key="index">{{data.restaurantName}}</option>
+            </select>
+          </td>
+        </tr>
+        <tr>
+          <td>
+            <label id="choiceLabel" for="secondChoice">Second Choice</label>
+          </td>
+          <td>
+            <select id="secondChoice" class= "form-control">
+              <option v-for="(data, index) in restaurantData" :key="index">{{data.restaurantName}}</option>
+            </select>
+          </td>
+        </tr>
+        <tr>
+          <td>
+            <label id="choiceLabel" for="thirdChoice">Third Choice</label>
+          </td>
+          <td>
+            <select id="thirdChoice" class= "form-control">
+              <option v-for="(data, index) in restaurantData" :key="index">{{data.restaurantName}}</option>
+            </select>
+          </td>
+        </tr>
+        <tr>
+          <td>
+            <label id="choiceLabel" for="fourthChoice">Fourth Choice</label>
+          </td>
+          <td>
+            <select id="fourthChoice" class= "form-control">
+              <option v-for="(data, index) in restaurantData" :key="index">{{data.restaurantName}}</option>
+            </select>
+          </td>
+        </tr>
+        <tr>
+          <td>
+            <label id="choiceLabel" for="fifthChoice">Fifth Choice</label>
+          </td>
+          <td>
+            <select id="fifthChoice" class= "form-control">
+              <option v-for="(data, index) in restaurantData" :key="index">{{data.restaurantName}}</option>
+            </select>
+          </td>
+        </tr>
+      </tbody>
+    </table>
+    <router-link to="/Home" tag="button" class="btn">Back</router-link>
+    <input class="btn btn-primary" type="submit" value="Submit" v-on:click="submitUpdate()" >
+    </div>
+  </div>
+</template>
+<script>
+  import axios from 'axios'
+  import VueOnToast from 'vue-on-toast'
+
+  export default {
+    data () {
+      return {
+        restaurantData: [],
+        topFiveData: [],
+        errors: []
+      }
+    },
+    created () {
+      axios.get('http://localhost:48146/lunchdata')
+        .then(response => {
+          console.log(response)
+          this.restaurantData = response.data
+          return this.restaurantData
+        })
+      axios.get('http://localhost:48146/topfivedata')
+        .then(response => {
+          console.log(response)
+          this.topFiveData = response.data
+          return this.topFiveData
+        })
+        .catch(e => {
+          this.errors.push(e)
+        })
+    },
+    methods: {
+      submitUpdate: function () {
+        VueOnToast.ToastService.pop('success', 'Successful Update', 'your preferences have been saved')
+      }
+    }
+  }
+</script>
+<style scoped>
```

Comment:
scoping the style takes a lot of horsepower behind the scenes (when the build processes the styles, especially if it's pulling them into a separate file, it's got to generate the appropriate css to make sure these all stay scoped to just this file). Consider changing this to just <style>, giving the top-level div a component-specific id, and nesting all the component-specific styles inside that id, e.g.:
#edit-top-five { .form-control { width: 100% } /*your styles*/  }